vnext/Desktop/ABI/MemoryTracker.idl

```diff
@@ -3,7 +3,7 @@ import "ABI/MessageQueue.idl";
 // ABI_REVIEW:
 // Given this is a new API without usage legacy, should we use the chance to introduce a different
 // namespace, e.g. "Microsoft.React"?
-namespace facebook.react {
+namespace Microsoft.React {
```

Comment:
>Microsoft.React  [](start = 10, length = 16)

While a meaningful change, is it a necessary part of the project restructuring?

Reply:
It's not. Already agreed to revert.